### PR TITLE
Updated requirements.txt with latest versions

### PR DESCRIPTION
**Pull request by vinodhini8694**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Flask==1.1.2
-pyarrow==3.0.0
-lakefs-client~=0.33.1
+Flask==2.2.2
+pyarrow==9.0.0
+lakefs-client~=0.81.1
+jinja2<3.1.2
```
Comment (nopcoder) [inline requirements.txt:4]: The code doesn't use jinja

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (vinodhini8694) [reply requirements.txt:4]: This is the error I got without jinja:

Traceback (most recent call last):
  File "/usr/local/bin/flask", line 5, in <module>
    from flask.cli import main
  File "/usr/local/lib/python3.9/site-packages/flask/__init__.py", line 14, in <module>
    from jinja2 import escape
ImportError: cannot import name 'escape' from 'jinja2' (/usr/local/lib/python3.9/site-packages/jinja2/__init__.py)

Comment (vinodhini8694) [reply requirements.txt]: @nopcoder  Was able to run hooks server without jinja. Updated the requirements.txt

Comment (nopcoder) [reply requirements.txt]: Flask got a list of dependencies (https://flask.palletsprojects.com/en/latest/installation/). But these will be installed as part of it's requirements, not ours.

**Review by nopcoder — APPROVED** [review]
lgtm!

**Pull request merged**